src/resources/i18n/am.json

```diff
@@ -116,5 +116,34 @@
     "patientregistrationsuccessfull": "The patient registration was successful.",
     "somethingwrong": "Something went wrong, please retry later.",
     "searchpatientinstruction": "If you don't have the patient ID, fill as many fields as possible and click search"
+  },
```

Comment:
There's no need to repeat these properties in every i18n resource file. The application uses English by default, so you just need to add these new strings to `en.json`.

Reply:
I understand.